lib/student_greeter.rb

```diff
@@ -0,0 +1,84 @@
+class StudentGreeter
+  def self.greet_all_ungreeted_students
+    new.greet_all_ungreeted_students
+  end
+
+  def initialize
+    @greeters = User.where(greeter: true)
```

Comment:
A greeter, at this point, is either of two users (Ian and Adam). They both are admins, but they are greeting in their capacity as the Wiki Ed Content expert assigned to a course. (There is usually also another staff member in the Wiki Ed Staff role for any given course, the Classroom Program Manager who is the instructor's point of contact, but that user is not a greeter.)

So, the greeter flag is intended to say 'this is one of the few users who will automatically greet students'. (In general, the system should not be making edits that don't directly result from an intentional action that the user just took. But staff accounts are the one exception.)